language-rust.cabal

```diff
@@ -41,7 +41,7 @@ library
                        -Wincomplete-uni-patterns
                        -Wmissing-signatures
 
-  build-tools:         alex >=3.1, happy >=1.19.8
+  build-tools:         alex >=3.1, happy >=1.19.8 && < 2.1
```

Comment:
Leave a comment here citing https://github.com/haskell/happy/issues/320 as the reason why the upper version bound doesn't allow `2.1` or later.